src/org/zaproxy/zap/extension/viewstate/zap/utils/ASPViewState.java

```diff
@@ -0,0 +1,209 @@
+package org.zaproxy.zap.extension.viewstate.zap.utils;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.regex.Pattern;
+
+import org.apache.log4j.Logger;
+import org.parosproxy.paros.extension.encoder.Base64;
+
+import net.htmlparser.jericho.Source;
+import net.htmlparser.jericho.StartTag;
+
+public class ASPViewState extends ViewState {
+
+	private static Logger logger = Logger.getLogger(ASPViewState.class);
+
+	private boolean isValid = true;
+	private static boolean isSplit = false;
```

Comment:
- This initialisation could be removed;
- Should it really be a class variable?